python/src/multi_agent_orchestrator/classifiers/bedrock_classifier.py

```diff
@@ -13,18 +13,25 @@ def __init__(
         self,
         model_id: Optional[str] = None,
         region: Optional[str] = None,
-        inference_config: Optional[Dict] = None
+        inference_config: Optional[Dict] = None,
+        aws_profile_name: Optional[str] = None
+
     ):
         self.model_id = model_id
         self.region = region
         self.inference_config = inference_config if inference_config is not None else {}
+        self.aws_profile_name = aws_profile_name
+        
 
 
 class BedrockClassifier(Classifier):
     def __init__(self, options: BedrockClassifierOptions):
         super().__init__()
         self.region = options.region or os.environ.get('REGION')
-        self.client = boto3.client('bedrock-runtime', region_name=self.region)
+        
+        boto3_session = boto3.Session(profile_name=self.aws_profile_name)
+        self.client = boto3_session.client('bedrock-runtime', region_name=self.region)
```

Comment:
Does this work if there is no profile_name mentioned?
I believe we should do something like this:

`if self.credentials_profile_name is not None:
    session = boto3.Session(profile_name=self.credentials_profile_name)
 else:
    session = boto3.Session()`